Makefile

```diff
@@ -616,18 +616,24 @@ remotesystem:
 	fi;\
 	exit $$rc
 
-.PHONY: localapiv2
-localapiv2:
-	# Order is important running python tests first causes the bash tests to fail, see 12-imagesMore
-	# FIXME order of tests should not matter
+.PHONY: localapiv2-1
+localapiv2-1:
```

Comment:
Can we name this more descriptive, i.e. `localapiv2-bash` and `localapiv2-python` instead

Reply:
Good idea, sure thing.